lib/italian.rb

```diff
@@ -0,0 +1,21 @@
+# Understands how to hold menu options
+require_relative "main"
+require_relative "basket"
+
+class Italian
```

Comment:
Italian sounds like a type of Menu, rather than Menu being a property of an Italian.